slack_sdk/web/client.py

```diff
@@ -77,12 +81,7 @@ def admin_analytics_getFile(
         **kwargs,
     ) -> SlackResponse:
         """Retrieve analytics data for a given date, presented as a compressed JSON file
-
-        Args:
```

Comment:
Now that we have all arguments, maintaining Args comment for 240+ API methods becomes more challenging. Instead, we can have a link to the API method document.